radis/io/dbmanager.py

```diff
@@ -514,6 +516,27 @@ def get_nrows(self, local_file):
             raise ValueError(engine)
         return nrows
 
+    def get_columns(self, local_file):
```

Comment:
As mentionned in another comment ; it would be more organized if this method was moved in [DataFileManager](https://github.com/radis/radis/blob/634a77c0da26b8d8b97741d38cbdffed5de4d649/radis/io/hdf5.py#L72)  (which exclusively deals with Files; not necessarily Database objects... that's the case for your function ) 


And keep a wrapper here : 

```python
def get_columns(self, local_file):
    return self.get_datafile_manager().get_columns(local_file)
```

Reply:
I've done the changes, can you review them?

Reply:
perfect for me, merging as soon as tests pass